server-common/src/main/java/org/apache/kafka/server/common/MetadataVersion.java

```diff
@@ -217,8 +217,11 @@ public enum MetadataVersion {
     // Add ELR related supports (KIP-966).
     IBP_3_9_IV1(22, "3.9", "IV1", true),
 
+    IBP_3_9_IV2(23, "3.9", "IV2", false),
```

Comment:
Could we add a comment that this is for bootstrapping TV?

Reply:
Sounds good. I also need to make a few other changes when I rebase, so be on the lookout for more soon :) 

I planned to make the update where we don't send features as part of the registration/api versions if the version range is 0-0. I need to do this because Colin's fix only changes the min version. It is safe because a version range 0-0 is essentially the same as not supporting the feature. We will have it here until the MV is marked as production ready.

Reply:
> I planned to make the update where we don't send features as part of the registration/api versions if the version range is 0-0. 

Is that for the old versions of the requests or both the old and the latest version?

Reply:
I think we can just do it regardless of the version of the request. I was planning to do it in Quorum/BrokerFeatures files, which would just be the controller/broker's own view of supported features. Alternatively I can do it in the request. 

cc: @cmccabe

Reply:
Looks like Colin already pushed this as part of the kraft.version PR. I will just resolve the conflicts and update the comment you mention.

Reply:
Should we add a new MV just for bootstrapping a new feature or just piggyback on an existing one (e.g., IBP_3_9_IV1)?

>  I planned to make the update where we don't send features as part of the registration/api versions if the version range is 0-0.

Colin's PR only skipped a finalized version 0 feature in ApiResponse. Do you plan to do that for supported features for registration/api? Range 0-0 can now happen since some of the v1 features are not stable yet.

Reply:
Hmmm-- I thought Colin's PR would cover what I need to cover since I didn't think his MV was stable yet either

Reply:
I've updated the PR to include the change for broker registration